config-linux.md

```diff
@@ -121,6 +121,9 @@ Each entry has the following structure:
 * **`uid`** *(uint32, OPTIONAL)* - id of device owner.
 * **`gid`** *(uint32, OPTIONAL)* - id of device group.
 
+If a `devices` contains duplicated devices with same `path`, the runtime MUST generate an error.
```

Comment:
How about this?
`If a file already exists at `path` that does not match the requested device, the runtime MUST generate an error.`
We don't care what runtime already did now, we need it to generate an error and return.